src/solo.cpp

```diff
@@ -134,6 +135,10 @@ void Solo::initialize()
   // Create the joint module objects
   joints_.set_motor_array(motors_, motor_torque_constants_, joint_gear_ratios_,
                           joint_zero_positions_, motor_max_current_);
+
+  // Set the maximum joint torque available
+  max_joint_torques_ = 0.99 * joints_.get_max_joint_torques().array();
```

Comment:
I don't understand why using a factor of `0.99` here. If the user specifies the `max_joint_torques` to some value, that's the value we should use.

If this is about the saftey mechanism, the safety mechanism should be changed to hit once `torque > max_torque` instead of comparing to equality of `torque >= max_torque` like we do right now.

Reply:
I think we would be more robust to simple numerical noise by doing so.